bitbots_body_behavior/launch/behavior.launch

```diff
@@ -5,7 +5,7 @@
         <arg name="sim" value="$(var sim)" />
     </include>
 
-    <include file="$(find-pkg-share bitbots_head_behavior)/launch/head_behavior.launch">
```

Comment:
Please remove the head_behavior package as well as the head_capsule in the body_behavior.